src/main/java/ch/wisv/events/EventsApplication.java

```diff
@@ -2,7 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.orm.jpa.EntityScan;
+import org.springframework.data.jpa.convert.threeten.Jsr310JpaConverters;
 
+// Jsr310JpaConverters.class is necessary for correctly persisting e.g. LocalDateTime objects
+@EntityScan(basePackageClasses = {EventsApplication.class, Jsr310JpaConverters.class})
```

Comment:
Nice. Apparently Hibernate 5 has native support, but for now this is a fine workaround.